packages/piavpn/piavpn.pacscript

```diff
@@ -0,0 +1,47 @@
+name="piavpn"
+breaks="${name}-git"
+repology=("project: piavpn")
+version="3.3-06906"
+url="https://installers.privateinternetaccess.com/download/pia-linux-${version}.run"
+description="Private Internet Access VPN Client"
+hash="75efbf335af99673f55c81ff5f7e70d66ced52913ace2baca12253e19d3e64cf"
+maintainer="WRM-42 <y8bsbahy@anonaddy.me>"
+
+prepare() {
+  sudo groupadd ${name}
+	sudo groupadd piahnsd
+}
+
+build() {
+  true
+}
+
+install() {
+  chmod +x pia-linux-${version}.run
+  ./pia-linux-${version}.run --noexec --target "${SRCDIR}/${name}"
+  sudo install -Dm755 "piafiles" "${STOWDIR}/${name}"
+  sudo install -Dm755 "installfiles" "${STOWDIR}/${name}/bin"
```

Comment:
you forgot to cd into the directory it extracted the files into also these install lines don't work if you want to copy folders and rename them you have to use cp.